src/components/album/AlbumPreview.js

```diff
@@ -25,28 +28,60 @@ const AlbumPreview = ({
       return;
     }
     //extract the album id
-    const albumId = match[1];
+    const spotifyAlbumId = match[1];
+
     //  Spotify album URL for embedding
-    const spotifyAlbumUrl = `https://open.spotify.com/embed/album/${albumId}`;
+    const spotifyAlbumUrl = `https://open.spotify.com/embed/album/${spotifyAlbumId}`;
     setSpotifyEmbedUrl(spotifyAlbumUrl);
   }, [spotifyUrl]);
-
+  //console.log("Children:", children.type);
   return (
-    <CardContent style={{ marginTop: '1rem', width: '100%', display: 'flex' }}>
-      <iframe
-        title="Spotify Web Player"
-        src={spotifyEmbedUrl}
-        width={width}
-        height={height}
-        frameBorder={frameBorder}
-        style={{ borderRadius: 8, ...style, border: 'none' }}
-        allow={allow}
-        {...props}
-      />
-      {children && (
-        <div style={{ width: '100%', display: 'flex' }}>{children}</div>
-      )}
-    </CardContent>
+    <>
+      <CardContent
+        style={{
+          marginTop: '1rem',
+          width: '100%',
+          display: 'flex',
+          flexDirection: contentLayout,
+          justifyContent: 'center',
+        }}
+      >
+        <iframe
+          title="Spotify Web Player"
+          src={spotifyEmbedUrl}
+          width={width}
+          height={height}
+          frameBorder={frameBorder}
+          style={{
+            borderRadius: 8,
+            ...style,
+            border: 'none',
+            display: 'flex',
+            flexDirection: contentLayout,
+            justifyContent: 'center',
+          }}
+          allow={allow}
+          {...props}
+        />
+
+        <div
+          style={{
+            width: '100%',
+            display: 'flex',
+            flexDirection: contentLayout,
+            justifyContent: 'center',
```

Comment:
Since this div will only have one child element, (whatever is rendered by `children`) keep in mind that the flex properties here only apply to those children. You can play around with the values in the dev console; but the main thing I want to point out that just because we want the items in `<CardContent>` to be either "column" or "row" depending on what the parent component passes in; doesn't necessarily mean we wnat that same value for within each of the children (iframe, this div). So maybe it doesn't make sense to pass in that same prop to all these children, vs hardcoding some values there.

Reply:
https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/119d9dce-90ec-4693-9cb4-b897d37fefa3